CHANGELOG.md

```diff
@@ -4,6 +4,21 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## [0.14.0] - 2024-01-12
+
+### Changed
+
+- Fixed Python typings for `ThresholdMessageKit.decrypt_with_shared_secret()` and `combine_decryption_shares_simple()` ([#84])
```

Comment:
Is #84 a breaking change for "compatibility" if it is purely related to the binding signature, and not the underlying method? If it isn't a breaking change, could this instead be in a `Fixed` block instead of in the `Changed` and put below the `Added` block?

Of course #83 is indeed a breaking change and should be left here, just channeling my inner Bogdan for changelogs format. 😄

Reply:
Indeed, it's a `Fixed` change. Fixed.